spec/requests/books_spec.rb

```diff
@@ -11,7 +11,7 @@
   end
 
   it 'returns all books' do
```

Comment:
Nice work on testing. There's a cleaner approach to this. Instead of just compiling every request on a books_spec you can separate them by request type. Here's a "I didn't test this if it's working" example

```ruby
RSpec.describe "/api/books", type: :request do
  subject! { get '/api/books' }
  
  it { expect(response.content_type).to eq('application/json; charset=utf-8') }
end 
```

That'll be in ur requests/books/index_spec.rb. Much cleaner no?

Reply:
I'll definitely try this! thanks!